src/Nethermind/Nethermind.Blockchain/Synchronization/FastBlocks/FastBlocksFeed.cs

```diff
@@ -524,7 +524,7 @@ private int InsertReceipts(FastBlocksBatch batch)
             var receiptSyncBatch = batch.Receipts;
             int added = 0;
             long? lastPredecessor = null;
-            List<(long, TxReceipt)> validReceipts = new List<(long, TxReceipt)>();
+            List<(Block, TxReceipt[])> validReceipts = new List<(Block, TxReceipt[])>();
```

Comment:
so, I was hoping that we will be able to optimize the fast blocks downloaded to avoid the Block loading for receipt - why do we need it entire block instead of just header or hash?

Reply:
Blocks are already there, I don't need to load them